test/exe/run_test.rb

```diff
@@ -73,9 +82,10 @@ def set_krane_run_expectations(new_args: {}, run_args: {})
   end
 
   def krane_run!(flags: '')
+    cmd_flags = "#{flags} --template #{TASK_TEMPLATE}".split
```

Comment:
Unconditionally adding `--template #{TASK_TEMPLATE}` seems wrong. I liked the pattern from the deploy test (` flags += ' -f /tmp' unless flags.include?('-f')`)

Reply:
Thanks Danny, that pattern is really a more correct way. Will use it